plugin/src/main/java/org/autorefactor/refactoring/Refactorings.java

```diff
@@ -413,4 +413,13 @@ public BadLocationException call() throws Exception {
             throw ex;
         }
     }
+
+    /**
+     * Gets the ASTRewrite rewrite object.
+     *
+     * @return the ASTRewrite rewrite
+     */
+    public ASTRewrite getRewrite() {

```

Comment:
I know :)
I will try to see if we can otherwise once the rest of the code is clearer to me.